sbin/build.sh

```diff
@@ -235,14 +236,18 @@ printJavaVersionString()
 
 removingUnnecessaryFiles()
 {
-  echo "Removing unneccessary files now..."
+  echo "Removing unnecessary files now..."
 
-  rm -rf cacerts_area
-
-  cd build/*/images || return
+  if [ "$USE_DOCKER" == "true" ] ; then
+     cd build/*/images || return
+  else
+     rm -rf cacerts_area
 
-  mv j2sdk-image "${OPENJDK_REPO_TAG}"
```

Comment:
why have you removed my fix? this was added so that the extracted tarballs have a useful name rather than `j2sdk-image`